src/org/zaproxy/zap/extension/directorylistv2_3/files/fuzzers/dirbuster/directory-list-2.3-big.txt

```diff
@@ -74249,6 +74253,7 @@ hclogo
 buton
 test3
 CVSROOT
+CVS
```

Comment:
(adding comment again to show properly in PR :/ )
It seems that this is already included (in line 6356).